sql/log_event.h

```diff
@@ -1322,7 +1322,8 @@ class Log_event
   static Log_event* read_log_event(IO_CACHE* file,
                                    const Format_description_log_event
                                    *description_event,
-                                   my_bool crc_check);
+                                   my_bool crc_check,
+                                   my_bool print_errors= 1);
```

Comment:
Any reason to have a default of "1" here (not "true")?

Reply:
I never noticed before, but I suppose when I define and declare a bool in the same statement, I use `1` and `0`, but when the type isn't with the variable, I use `true` and `false` (as seeing in the function call I write `false`). Seems more concise that way to me, but I suppose that's up to preference, and it isn't in our coding guidelines.

Do you have a strong preference that it should be `true`/`false`?

Reply:
No strong preference, as you like. Just noticed that you were using 1 here and false the other place. Not important.